packages/gcf-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gcf-utils",
-  "version": "11.3.0",
+  "version": "11.3.0-debug.2",
```

Comment:
@SurferJeffAtGoogle the fact that I can do this is very useful, but something we should think about when we discuss BCID, I published a debugging version of client library to npm, which you have to opt into.